### PR TITLE
Iss 12 schemat bazy danych 

### LINKED ISSUE #12: schemat bazy danych

### PR DESCRIPTION
**Pull request by Hamilkar247**
fixes #12

### CODE REVIEW

File: api/templates/api/base.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!-- templates/front/base.html -->
```
Comment (jlatusek) [inline api/templates/api/base.html:1]: Why do you want to create templates view in api aplication? 
Do you think that this app should have own view? Why?

File: api/urls.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.urls import path, include
+from . import views
+
+urlpatterns = [
+#    path('', views.ZASTAP, name="ZASTAP"),
```
Comment (jlatusek) [inline api/urls.py:5]: Remove comment

File: core/urls.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.urls import path, include
+from . import views
+
+urlpatterns = [
+    #path('', views.ZASTAP, name="ZASTAP"),
```
Comment (jlatusek) [inline core/urls.py:5]: Remove

File: core/models.py [hunk 4]
```diff
@@ -1,3 +1,60 @@
 from django.db import models
 
-# Create your models here.
+
+class Deck(models.Model):
+    name = models.CharField(max_length=100, verbose_name="name of deck")
+    description = models.CharField(max_length=1200, verbose_name="Description of deck")
+    tag = models.CharField(max_length=50, verbose_name="tag")
```
Comment (jlatusek) [inline core/models.py:7]: https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ManyToManyField.through
Please add many to many field with trough field corresponded to DeckFlashCard

**Review by jlatusek — CHANGES_REQUESTED** [review]

File: api/templates/api/base.html (hunk 1 above)
Comment (Hamilkar247) [reply api/templates/api/base.html:1]: deleted

File: api/urls.py (hunk 2 above)
Comment (Hamilkar247) [reply api/urls.py:5]: deleted

File: core/urls.py (hunk 3 above)
Comment (Hamilkar247) [reply core/urls.py:5]: deleted

File: core/models.py (hunk 4 above)
Comment (Hamilkar247) [reply core/models.py:7]: i added but I have problem
AttributeError at /admin/core/deckflashcard/add/
'DeckFlashCard' object has no attribute 'deck'

Can you test create deck on your computer?

Comment (Hamilkar247) [reply core/models.py:7]: on admin panel

Comment (jlatusek) [reply core/models.py:7]: Fixed

**Comment by Hamilkar247** [comment]
Ok i check code

**Pull request merged**